simcon_project/templates/template_start_view.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
Not sure I totally understand what you're getting at. Can't I just merge in the actual page whenever it gets written and then there'd be no danger of me overwriting anything with this placeholder? Removing this would break a piece of my page.

Reply:
thought this was not one of ur pages